pandas/core/tools/datetimes.py

```diff
@@ -853,6 +853,17 @@ def to_datetime(
     ...                origin=pd.Timestamp('1960-01-01'))
     DatetimeIndex(['1960-01-02', '1960-01-03', '1960-01-04'], \
 dtype='datetime64[ns]', freq=None)
+
+    In case input is list-like and the elements of input are of mixed
+    timezones, return will have object type Index if utc=False.
+
+    >>> pd.to_datetime(['2018-10-26 12:00 -0530', '2018-10-26 12:00 -0500'])
+    Index([2018-10-26 12:00:00-05:30, 2018-10-26 12:00:00-05:00], dtype='object')
+
+    >>> pd.to_datetime(['2018-10-26 12:00 -0530', '2018-10-26 12:00 -0500'],
+    ...                utc=True)
+    DatetimeIndex(['2018-10-26 17:30:00+00:00', '2018-10-26 17:00:00+00:00'], \
+dtype='datetime64[ns, UTC]', freq=None)
```

Comment:
does this render properly?

Reply:
Sorry about that. Added a fix for the rendering. 
It now renders like below -

![image](https://user-images.githubusercontent.com/1128508/115491822-ab3c8e80-a22e-11eb-8084-36cb17853d7d.png)